tests/listsAndDropDowns.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { test, expect } from '@playwright/test';
+
+
+test.beforeEach(async({page}) => {
+    await page.goto('https://petclinic.bondaracademy.com/welcome')
+})
+
+test.describe('Lists and Dropdowns', () => {
+    test.beforeEach(async ({page}) => {
+        await page.getByRole('button', {name: ' Owners'}).click() //1. Select owners in nav bar and then click search
+        await page.getByRole('link', {name: ' Search'}).click()
+    })
+test('Validate selected pet types from list', async ({page}) => {
+    const petTypeMenuDropDown = page.locator('#type')
+    await page.goto('https://petclinic.bondaracademy.com/owners')
```

Comment:
I added the second `goto()` method because it seemed like sometimes the test would continue to run faster than the owners page would load. This was my solution to ensure the page was fully loaded before the next steps would execute. I have removed it.

Reply:
In situations like that should be a different approach, for example creating the Locator Assertion for some desired element to be displayed or adding the other versions of dynamic wait